DEVELOP.rst

```diff
@@ -7,6 +7,8 @@ Prerequisites
 
 Crate is written in Java_ 7, so a JDK needs to be installed. On OS X we
 recommend using `Oracle's Java`_ and OpenJDK_ on Linux Systems.
+Crate requires Java 7 update 55 or later or Java 8 update 20 or later.
```

Comment:
do previous java 8 version also suffer from data corruption? or why do we require java 8 u20 ?